crates/extensions/tedge_config_manager/src/tests.rs

```diff
@@ -133,6 +136,35 @@ async fn spawn_config_manager_actor(
     (mqtt, fs, downloader, uploader)
 }
 
+#[tokio::test]
+async fn default_plugin_config() {
+    let tempdir = TempTedgeDir::new();
+    let (_mqtt, _fs, _downloader, _uploader) = spawn_config_manager_actor(tempdir.path()).await;
+    let plugin_config_content =
+        read_to_string(tempdir.path().join("tedge-configuration-plugin.toml")).unwrap();
+    let plugin_config_toml: Table = from_str(&plugin_config_content).unwrap();
+
+    let tedge_config_path = format!("{}/tedge.toml", tempdir.path().to_string_lossy());
+    let tedge_log_plugin_config_path = format!(
+        "{}/plugins/tedge-log-plugin.toml",
+        tempdir.path().to_string_lossy()
+    );
+    let expected_config = toml::toml! {
```

Comment:
I don't think this test is helpful: the expected_config is built using exactly the same code as by the unit under test.

Reply:
I added this test just to make sure that any future changes to the default config are also always tested. Since the paths used are dynamic, I had no choice but to use a similar mechanism to generate the expected configs as well.